### PR TITLE
upgrading spring boot to 3.3.5, removing healthcheck from lb

### PR DESCRIPTION
**Pull request by foman-ch**
- upgrade Spring Boot to 3.5.5
- remove healthcheck path from load balancer

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: pom.xml
Comment (dhawes-ch) [inline pom.xml:1]: Might be worth replacing the log4j dependency management section with a spring starter logging dependency to align with the structured logging recommended approach?

If you agree, worth also bumping up structured logging to the latest which is 3.0.20 😛

**Review by aduffy-ch — APPROVED** [review]

File: pom.xml
Comment (foman-ch) [reply pom.xml:1]: taking a look, thanks :)

**Pull request closed without merging**